project/src/pages/page404/page404.tsx

```diff
@@ -0,0 +1,58 @@
+import { Link } from 'react-router-dom';
+import { AppRoute } from '../../const';
+
+import './page404.css';
+
+function Page404() {
+  return (
+    <div className="page page--not-found">
+      <header className="header">
+        <div className="container">
+          <div className="header__wrapper">
+            <div className="header__left">
+              <Link className="header__logo-link" to={AppRoute.Root}>
+                <img className="header__logo" src="img/logo.svg" alt="6 cities logo" width="81" height="41" />
```

Comment:
```suggestion
                <img 
                  className="header__logo" 
                  src="img/logo.svg" 
                  alt="6 cities logo" 
                  width="81" 
                  height="41" 
                />
```

там где больше одного пропса, лучше писать вот так.